### ISSUE TITLE
[feature request] Non-debian docker image

### ISSUE THREAD

**Issue by jerheij** [pi-hole/docker-pi-hole#362]
I have a short question about the base imaged used for pi-hole. I've noticed this is an Debian image and I am trying to run as much containers on my physical CentOS server instead of on my Debian VM. 

Is it possible to have non-Debian based (preferably Alpine) images with pi-hole again or are there limitations as of why this is not an option?

**Comment by nxadm** [comment]
The Debian stretch-slim base image is [around 22 MB](https://hub.docker.com/r/library/debian/tags/). While alpine may shave 20MB, I am pretty sure you'll get them when adding dependencies.

**Comment by jerheij** [comment]
> The Debian stretch-slim base image is [around 22 MB](https://hub.docker.com/r/library/debian/tags/). While alpine may shave 20MB, I am pretty sure you'll get them when adding dependencies.

It wasn't as much about size as conpatibility. At least; that's what I previously thought. For some reason I didn't got Debian based images to work on the docker running on CentOS so I stopped looking into it and made a (stupid) assumption that Debian containers on non-Debian based OS is a no-go.

After some more trial and error on my side it did seem to work. Maybe it had to do with a wrong Docker setup or something else but on a more freshly installed CentOS 7 machine I did get it to work and migrated pihole successfully from my Debian docker installation to my CentOS docker installation.

Thanks anyway!
